subsys/logging/log_backend_console.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <logging/log_backend_console.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <device.h>
+#include <uart.h>
+
+int char_out(u8_t *data, size_t length, void *ctx)
+{
+	struct device *dev = (struct device *)ctx;
+
+	for (size_t i = 0; i < length; i++) {
+		uart_poll_out(dev, data[i]);
+	}
+	return length;
+}
+
+static u8_t buf;
+
+static struct log_output_ctx ctx = {
+	.func = char_out,
+	.data = &buf,
+	.length = 1,
+	.offset = 0
+};
+
+static void put(struct log_backend const *const backend,
```

Comment:
I don't see anything "console"-specific in this function. Why don't name it log_generic_put(), move it to a common file and allow different backends to reuse?